<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Materials</class>
 <widget class="QWidget" name="Materials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Материалы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelItemName">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditItemName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelUnit">
       <property name="text">
        <string>Единица измерения</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxUnits">
       <item>
        <property name="text">
         <string>шт.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>пачка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>кг.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>коробка</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelItemPrice">
       <property name="text">
        <string>Стоимость</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxItemPrice">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonItemInsert">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutEditing">
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdateCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
